iree-jax/benchmark/benchmark_model.py

```diff
@@ -52,18 +56,41 @@ def dump_result(file_path: str, result: dict) -> None:
 def bytes_to_mb_str(bytes: Optional[int]) -> str:
   return "n/a" if bytes is None else f"{bytes / 1e6:.6f}"
 
-
-def run_framework_benchmark(model_name: str, model_class: Any, batch_size: int,
+def retrieve_data(model_data: data_types.ModelData, cache_dir: str) -> tuple[Any, ...]:
+  data = ()
+  for url in model_data.source_url:
+    assert url.startswith("https://storage.googleapis.com/iree-model-artifacts/")
+    relative_path = url.removeprefix("https://storage.googleapis.com/iree-model-artifacts/")
+    local_path = cache_dir / relative_path
+
+    if not os.path.exists(local_path):
```

Comment:
```suggestion
    if not local_path.exists():
```

Reply:
I don't think this works since `local_path` is a string.

Reply:
Oh my mistake, it works. I've created a new PR that refactors this logic into a utils file.